README.md

```diff
@@ -50,6 +50,10 @@ npm install chrome-launcher
   // * Otherwise, a detected Chrome (stable) will be used
   chromePath: string;
 
+  // (optional) Uses the default Chrome profile instead of creating a new one
+  // Default: false
+  defaultProfile: boolean;
```

Comment:
to 🚲 🏠  on name, it's not super clear that this will currently ignore whatever is set as `userDataDir`

some potential others

* `forceDefaultProfile`
* `ignoreUserDataDir`
* `overrideUserDataDirWithDefault`

alternatively, we could make this the behavior when `false` is explicitly passed to `userDataDir` and just document that it results in the default profile being used?

Reply:
I like `forceDefaultProfile`. 

> alternatively, we could make this the behavior when `false` is explicitly passed to `userDataDir` and just document that it results in the default profile being used?

Do you want me to add this behavior on top of the current one or instead of the new parameter? `userDataDir` will probably be a cleaner API for the user, but I was looking into the code and I had to start changing where the `pid` files were created, etc. It was starting to get a bit too complicated so I preferred to wait for feedback.

Reply:
> Do you want me to add this behavior on top of the current one or instead of the new parameter? 

My personal preference would probably be the boolean approach instead of the new flag. I agree it's a cleaner API for consumers. OTOH, `forceDefaultProfile` is fairly explicit while `userDataDir: false` is a little more opaque even if the logic tracks of "do not pass in --user-data-dir".

> userDataDir will probably be a cleaner API for the user, but I was looking into the code and I had to start changing where the pid files were created, etc. It was starting to get a bit too complicated so I preferred to wait for feedback.

Hmmm, I suppose there's not a clear and obvious alternative for where else to stick the pid file `process.cwd()` might be ok but permissions and litter could get messy. Keeping the tmp directory for now seems ok to me.

Reply:
> OTOH, `forceDefaultProfile` is fairly explicit while `userDataDir: false` is a little more opaque even if the logic tracks of "do not pass in --user-data-dir".

Been trying to do this and there are some things to take into account. The main problem is "what happens if the users puts `true` instead of a `path` or `false`? Also I need to update the logic in a couple places to validate that `userDataDir` is a `string` instead of `boolean`, otherwise TypeScript will complain.

This is my current code in the constructor:

```js
if (typeof this.opts.userDataDir === 'boolean') {
      if (!this.opts.userDataDir) {
        this.forceDefaultProfile = true;
      } else {
        throw new Error('userDataDir has to be false or a path');
      }
    } else {
      this.forceDefaultProfile = defaults(this.opts.forceDefaultProfile, false);
    }
  }
```

What do you think? Should I just add the `forceDefaultProfile` parameter or continue down this path?